tests/test_utils.py

```diff
@@ -987,23 +987,23 @@ def test_convert_params(input_: Any, expected: Any, convert_mode: Any):
     ],
 )
 def test_convert_params_with_dataclass_obj(input_: Any, expected: Any, convert: Any):
-    # Instantiated dataclasses are never converted to primitives.
-    # This is due to the ambiguity between dataclass as an object and as
-    # an input for creating a config object (Structured Configs)
-    # Even in ConvertMode.ALL - the underlying dict and list are converted to DictConfig and ListConfig
-    # because the parent DictConfig cannot hold unwrapped list and dict.
-    # This behavior is unique to instantiated objects that are dataclasses.
+    """Test instantiation of dataclass instances.DictConfig
 
+    Except for when convert = ConvertMode.ALL, dataclass instances are
+    treated as OmegaConf DictConfig.
+    """
     cfg = OmegaConf.create(input_)
-    kwargs = {"a": {"_convert_": convert}}
```

Comment:
If `_convert_` is only applied to `"a"`, it does not work (currently).

The question is : if a top-level node has `convert = convert.None`, does it supersede all children convert modes?